code-challenges.test.js

```diff
@@ -15,24 +15,83 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+describe("Colors Array", () => {
+    it("removes the first item from the array and shuffles the remaining items", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+        const colorOutput1 = ["yellow", "blue", "pink", "green"]
+        const colorOutput2 = ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+        expect(colors1).toEqual(expect.arrayContaining(colorOutput1))
+        expect(colors2).toEqual(expect.arrayContaining(colorOutput2))
+    }) 
+})
+
+
 
 // b) Create the function that makes the test pass.
 
+const removeFirstAndShuffle = function(arr) {
+    if (arr.length > 1) {
+      // If the array has more than one item
+      const firstItemRemoved = arr.slice(1)
+       // Remove the first item
+  
+      // Shuffle the items randomly
```

Comment:
Nice work in this section!  You could expand a little more on your pseudocode specifically for line 44 and talk about how this is randomly shuffling the items.